alb/terraform-examples/nsxt-cloud-data-segment/Create Subsequent Data Segments.md

```diff
@@ -0,0 +1,194 @@
+# NSXT Cloud Data Subsequent Segment Creation
+
+Scripts Developed by: William Stoneman</br>
+
+# Introduction
+
+The purpose of this file is to lay out the steps to create subsequent NSX-T Data Segment after the NSX-T Cloud Connector has been successfully created and the intial Data Segment is configured. 
+
+The information provided below is for demonstration purposes only, and you should update the variables with the information from your environment.
+
+# Low Level Process Review
+
+
+1. Create tf script:
+
```

Comment:
Can the script content be placed in code blocks with hcl destination?

Example:

```hcl
terraform {}
```